.travis.yml

```diff
@@ -4,10 +4,6 @@ sudo: false
 
 matrix:
   include:
-    - php: 7.2
```

Comment:
Why are not checking any php version?

We are just checking vs PHP 8

If travis is not working, we could try github actions.